services/gateway/config.js

```diff
@@ -80,6 +80,15 @@ config.api.enableHttpCacheControl = Boolean(String(process.env.ENABLE_HTTP_CACHE
 // Unless STRICT_READINESS_CHECK env. variable is set false, includeCoreReadiness evaluates to true
 config.includeCoreReadiness = Boolean(String(process.env.STRICT_READINESS_CHECK).toLowerCase() !== 'false');
 
+// configuration for websocket rate limit
+config.websocket = {
+	enableRateLimit: Boolean(String(process.env.ENABLE_WS_RATELIMIT).toLowerCase() !== 'false'),
```

Comment:
Let us disable `enableRateLimit` until and unless `ENABLE_WS_RATELIMIT` is explicitly set to `true`.

```suggestion
	enableRateLimit: Boolean(String(process.env.ENABLE_WS_RATELIMIT).toLowerCase() === 'true'),
```